include/mc_tasks/ImpedanceTask.h

```diff
@@ -227,10 +227,18 @@ struct MC_TASKS_DLLAPI ImpedanceTask : TransformTask
   /** Targets of SurfaceTransformTask should not be set by the user.
    *  Instead, the user can set the targetPose, targetVel, and targetAccel.
    *  Targets of SurfaceTransformTask are determined from the target values through the impedance equation.
+   *
+   *  We override the target functions of TransformTask to set the targetPose instead.
+   *  This allows functions such as targetFrame, targetSurface to work as expected.
    */
   using TransformTask::refAccel;
   using TransformTask::refVelB;
-  using TransformTask::target;
+
+  /* \brief Same as targetPose(const sva::PTransformd &) */
+  void target(const sva::PTransformd & pos) override { targetPose(pos); }
+
+  /* \brief Same as targetPose() */
+  sva::PTransformd target() const override { return targetPose(); }
```

Comment:
Oh, sorry. I understand that this is virtual, so it has a meaning when called from inside TransformTask. I understand.

Reply:
Indeed, I in fact considered making it public anyways for consistency between `TransformTask` and `ImpedanceTask`, but since we already had `ImpedanceTask::targetPose` being widely used I left it private. Plus having to explicitly use `targetPose` highlights that the impedance task target is different from the underlying `TransformTask`.